client/state/data-layer/wpcom/comments/index.js

```diff
@@ -156,6 +160,35 @@ export const announceFailure = ( { dispatch, getState }, { siteId, postId } ) =>
 	dispatch( errorNotice( error ) );
 };
 
+export const deleteComment = ( { dispatch }, action ) => {
+	const { siteId, commentId } = action;
+	dispatch(
+		http( {
+			method: 'POST',
+			apiVersion: '1.1',
+			path: `/sites/${ siteId }/comments/${ commentId }/delete`,
+			onSuccess: {
+				...action,
+				type: COMMENTS_DELETE_SUCCESS,
+			},
+			onFailure: {
+				...action,
+				type: COMMENTS_DELETE_FAILURE,
+			},
+		} )
+	);
+};
+
+export const removeCommentFromState = ( { dispatch }, action ) => {
+	const { siteId, postId, commentId } = action;
+	dispatch( removeComment( siteId, postId, commentId ) );
+};
+
+export const announceDeleteFailure = ( { dispatch } ) => {
+	dispatch( errorNotice( translate( 'Could not delete the comment' ) ) );
```

Comment:
Hi! I've found a possible matching string that has already been translated **9** times:
`translate( 'Sorry, you can not delete this comment.' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).